src/Http/Controllers/VoyagerDatabaseController.php

```diff
@@ -59,6 +56,40 @@ public function create()
         return Voyager::view('voyager::tools.database.edit-add', compact('db'));
     }
 
+    protected function prepareDbManager($action, $table = '')
```

Comment:
Any particular reason this method was moved?

Reply:
PHPStorm auto format on save